h2o-docs/src/product/migrating.rst

```diff
@@ -1,4 +1,30 @@
 .. _migration:
+.. include:: <isonum.txt>
+
+Migration Guide
+###############
+
+Migrating between H2O-3 versions
+================================
+
+H2O-3 does its best to keep backwards compatibility between major versions, but sometimes breaking changes are needed in order to improve code quality and to address issues.
+
+Please refer to the following sections if you encounter any API-related issue when upgrading H2O-3.
+
+**Note**: H2O-3 versions follow the naming convention ``3.major.minor.patch``. 
+
+From 3.22 or below to 3.24
+--------------------------
+
+Java API
+~~~~~~~~
+
+The following classes were moved and/or renamed:
+
+- ``hex.StackedEnsembleModel`` |rarr| ``hex.ensemble.StackedEnsembleModel``.
+- ``hex.StackedEnsembleModel.MetalearnerAlgorithm`` |rarr| ``hex.ensemble.Metalearner.Algorithm``.
+
```

Comment:
What is |rarr| in the above two bullets? These references are causing an error when building the docs.

Reply:
@angela0xdata those are non-ascii characters (right-arrow in this case), hence the `isonum` import above.
I built the docs locally before and it worked fine, that's why I included it... maybe it depends on the environment though. If it doesn't work everywhere, I'll remove it.

Reply:
for reference: http://docutils.sourceforge.net/docs/ref/rst/definitions.html